test/truffle/cexts/unf_ext/unf_ext.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+source test/truffle/common.sh.inc
+
+jt gem-test-pack
+
+export TRUFFLERUBY_CEXT_ENABLED=true
+jt ruby -S gem install --local "$(jt gem-test-pack)/gem-cache/unf_ext-0.0.7.4.gem" -V -N --backtrace
```

Comment:
:+1: This is much more useful than tweaking our internal cextc/compile_cext :)

Reply:
Well, I just updated jt.rb, which already has a whole bunch of gem-specific hacks in it ;-)

Reply:
And less hacks is better than more hacks :)